src/app/components/author/author.component.html

```diff
@@ -1,5 +1,5 @@
 @if (author) {
-  <div
+  <a
```

Comment:
A better approach would be handling the link in the parent component. We can remove anything related to the profile URL and keep it as a `div`. Then, when this needs to be a link, the smart component can wrap it in a link tag.